### ISSUE TITLE
ubuntu 22.04 l2tp vpn 

### ISSUE THREAD

**Issue by hamedg68** [nm-l2tp/NetworkManager-l2tp#190]
I installed ubuntu 22.04 and configed l2tp vpn, after l2tp connected my internet connection broken, why???
I have tried xl2pd versions (1.3.16-1ubuntu0.1 amd64) and (1.3.12-1.1) but none of them work for me.
after many search I can not find solution.
can anyone help me?

**Comment by dkosovic** [comment]
I'm assuming it was working for you before with an earlier version of Ubuntu, if so, the bug is probably NetworkManager 1.36 introducing a spurious route and in some cases spurious IP address, see later messages in the upstream bug report: 
* https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/946

Otherwise, I would need to see the log output with:
```
sudo journalctl --no-hostname _SYSTEMD_UNIT=NetworkManager.service + SYSLOG_IDENTIFIER=pppd
```

change any IP address to anonymise it if you wish.

**Comment by dkosovic** [comment]
I'll assume you were experiencing the aforementioned upstream NetworkManager 1.36 bug with spurious route and/or spurious IP address.

I'll close this issue as there is not much I can do, they need to fix the upstream NetworkManager 1.36 bug.
